src/translations/en.json

```diff
@@ -1772,7 +1772,22 @@
             "title": "Logs"
           },
           "events": {
-            "title": "Events"
+            "title": "Events",
+            "description": "Fire an event on the event bus.",
+            "documentation": "Events Documentation.",
+            "type": "Event Type",
+            "data": "Event Data (YAML, optional)",
+            "fire_event": "Fire Event",
+            "event_fired": "Event {name} fired",
+            "available_events": "Available Events",
+            "count_listeners": " ({count}  listeners)",
+            "listen_to_events": "Listen to events",
+            "listening_to": "Listening to",
+            "subscribe_to": "Event to subscribe to",
+            "start_listening": "Start listening",
+            "stop_listening": "Stop listening",
+            "alert_event_type": "Event type is a mandatory field",
+            "notification_event_fired": "Event {type} successful fired!"
```

Comment:
I don't see this one in your code?

Reply:
Oh my bad. Not sure why I did not use `start/stop_listening`. Though I wanted to use `alert_event_type` and `notification_event_fired` in developer-tools-event.js here:
```javascript
fireEvent() {
    if (!this.eventType) {
      alert("Event type is a mandatory field");
      return;
    }
    this.hass.callApi("POST", "events/" + this.eventType, this.parsedJSON).then(
      function() {
        this.fire("hass-notification", {
          message: "Event " + this.eventType + " successful fired!",
        });
      }.bind(this)
    );
  }
```
However, could not figure out how to use `this.hass.localize()` in this contex, the frontend always showed blank instead of the translated text.
Is there anything I have to take into account?

Reply:
should work, what was your code?

Reply:
Not sure what I did wrong before, but now it works flawlessly. Updated my code with 923cfe7db40bb26d8fe2e34507cab79da889b241.